src/Admin/AdminConsole/Controllers/ProvidersController.cs

```diff
@@ -213,6 +219,13 @@ public async Task<IActionResult> Edit(Guid id, ProviderEditModel model)
         {
             foreach (var providerPlan in providerPlans)
             {
+                providerPlan.SeatMinimum = providerPlan.PlanType switch
+                {
+                    PlanType.TeamsMonthly => model.TeamsMonthlySeatMinimum,
+                    PlanType.EnterpriseMonthly => model.EnterpriseMonthlySeatMinimum,
+                    _ => providerPlan.SeatMinimum
+                };
```

Comment:
🎨  This is a style nit only, and fairly subjective, so it's ultimately your decision: I think the switch statement makes this harder to read than just if/else, and it makes you handle a redundant default case (`providerPlan.SeatMinimum = providerPlan.SeatMinimum`). (I know this is often a suggestion by Rider, I just disagree with it.)

Reply:
Changed to `if/else`.